src/relay.rs

```diff
@@ -170,6 +172,12 @@ impl RelayMap {
     pub fn pop_accepted_connect_request(&mut self, endpoint: &Endpoint) -> Option<PublicId> {
         self.accepted_connect_requests.remove(endpoint)
     }
+
+    /// Returns true if the relay map was instantiated with a self_relocated id.
+    /// A self_relocated id should only be used by the first node to start a network.
+    pub fn zero_node(&self) -> bool {
```

Comment:
Just a suggestion, but I would prefer if this was called `is_self_relocated` instead of introducing new terminology.